Library/OcBootManagementLib/BootEntryInfo.c

```diff
@@ -698,13 +698,15 @@ InternalDescribeBootEntry (
     BootEntry->Name = AllocateCopyPool (sizeof (L"Windows"), L"Windows");
   }
 
+  // TODO: Should macOS installer have own OC_BOOT_ENTRY_TYPE (plus own voiceover file?)?
```

Comment:
Think yes…

Reply:
I think yes, too, but it's another bunch of fiddly changes... not entirely sure if it would happen for Monday or similar timescale.

Reply:
Can postpone, yes.